app/templates/_package.json

```diff
@@ -58,6 +60,8 @@
 		"karma-coverage": "^0.5.1",
 		"karma-mocha": "^0.2.0",
 		"karma-source-map-support": "1.0.0",
+		"karma-html2js-preprocessor": "^1.0.0",
```

Comment:
This and `metal-karma-config` are not used everywhere, so it's best to remove them to keep the dependencies as small as possible. We like to keep even the dev dependencies minimal, covering just the essencial for most cases, as they already take a long time to download as it is.